packages/react-components/react-card/e2e/Card.e2e.tsx

```diff
@@ -0,0 +1,243 @@
+import * as React from 'react';
+import { mount } from '@cypress/react';
+import type {} from '@cypress/react';
+import { FluentProvider } from '@fluentui/react-provider';
+import { webLightTheme } from '@fluentui/react-theme';
+import { Button } from '@fluentui/react-button';
+import { Card, CardFooter, CardHeader } from '@fluentui/react-card';
+import type { CardProps } from '@fluentui/react-card';
+
+const mountFluent = (element: JSX.Element) => {
+  mount(<FluentProvider theme={webLightTheme}>{element}</FluentProvider>);
+};
+
+const CardSample = (props: CardProps) => {
+  const ASSET_URL = 'https://raw.githubusercontent.com/microsoft/fluentui/master/packages/react-card';
+
+  const powerpointLogoURL = ASSET_URL + '/assets/powerpoint_logo.svg';
+
+  return (
+    <>
+      <p tabIndex={0} id="before">
+        Before
+      </p>
+      <Card id="card" {...props}>
+        <CardHeader
+          image={<img src={powerpointLogoURL} alt="Microsoft PowerPoint logo" />}
+          header={<b>App Name</b>}
+          description={<span>Developer</span>}
+        />
+        <div>
+          Donut chocolate bar oat cake. Dragée tiramisu lollipop bear claw. Marshmallow pastry jujubes toffee sugar
+          plum.
+        </div>
+        <CardFooter>
+          <Button id="open-button" onClick={alert} appearance="primary">
+            Open
+          </Button>
+          <Button id="close-button" appearance="outline">
+            Close
+          </Button>
+        </CardFooter>
+      </Card>
+      <p tabIndex={0} id="after">
+        After
+      </p>
+    </>
+  );
+};
+
+describe('Card', () => {
+  describe('focus behaviors', () => {
+    describe('focusMode="off" (default)', () => {
+      it('should not be focusable', () => {
+        mountFluent(<CardSample />);
+
+        cy.get('#before').focus();
```

Comment:
nit: you seem to use these selectors multiple times, would be worth hoisting them to variables

Reply:
The duplication is intentional for readability purposes as hardcoded values makes the test clearer (less need to hop around to understand what it is doing).